README.md

```diff
@@ -1,12 +1,16 @@
-<p align="center"><img src="https://flarum.org/img/logo.png"></p>
+<p align="center"><img src="https://discuss.flarum.org/assets/logo-2q2rcevl.png"></p>
 
 <p align="center">
-<a href="https://travis-ci.org/flarum/core"><img src="https://travis-ci.org/flarum/core.svg" alt="Build Status"></a>
+<a href="https://github.com/flarum/core/actions"><img src="https://github.com/flarum/core/workflows/JavaScript/badge.svg" alt="Git Actions"></a>
```

Comment:
This one is just the workflow that builds our JS assets - we probably don't need a badge for that one.

Reply:
The badges have been removed. Thank you